include/zephyr/devicetree.h

```diff
@@ -2534,6 +2623,26 @@
 #define DT_INST_STRING_UPPER_TOKEN(inst, prop) \
 	DT_STRING_UPPER_TOKEN(DT_DRV_INST(inst), prop)
 
+/**
+ * @brief Get an element out of string-arrap property as a token.
```

Comment:
Get an element out of a string-array property as a token.

Reply:
Looks like there is a typo in the latest push: `string-array`, not `string-arrap`